tests/test_cases/test_multi_dimension_array/Makefile

```
@@ -27,24 +27,35 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-ifeq ($(SIM),)
-all:
-	@echo "Skipping test_multi_dimension_array since icarus doesn't support indexing"
 
-clean::
-# nothing to clean, just define target in this branch
+ifeq ($(SIM),)
+USING_ICARUS := 1
+endif
 
-else ifeq ($(shell echo $(SIM) | tr A-Z a-z),icarus)
-all:
-	@echo "Skipping test_multi_dimension_array since icarus doesn't support indexing"
+ifeq ($(shell echo $(SIM) | tr A-Z a-z),icarus)
+USING_ICARUS := 1
+endif
 
-clean::
-# nothing to clean, just define target in this branch
+ifneq ($(USING_ICARUS),)
+ICARUS_MIN := 11.0
+ICARUS_VERSION := $(shell iverilog -V 2>/dev/null | head -n1 | cut -d ' ' -f 4)
+MIN_VERSION := $(shell printf "%s\n%s\n" "$(ICARUS_MIN)" "$(ICARUS_VERSION)" | sort -g | head -1)
+ifneq ($(MIN_VERSION),$(ICARUS_MIN))
+SKIP := 1
+$(info Skipping test_multi_dimension_array since icarus < v$(MIN_VERSION) doesn't support packages)
```

?
CI log is currently showing `Skipping test_multi_dimension_array since icarus < v10.3 doesn't support packages`
```suggestion
$(info Skipping test_multi_dimension_array since icarus < v$(ICARUS_MIN) doesn't support packages)
```